corehq/apps/accounting/utils.py

```diff
@@ -214,6 +215,8 @@ def get_customer_cards(username, domain):
         return stripe_customer.cards
     except StripePaymentMethod.DoesNotExist:
         pass
+    except stripe.error.AuthenticationError:
```

Comment:
I wonder if this might catch some errors on which we'd want to fail hard.  Would it be better to just return ```None``` if ```not settings.STRIPE_PRIVATE_KEY``` rather than relying on catching an exception?